travis/clone_oca_dependencies

```diff
@@ -60,6 +60,11 @@ def git_checkout(deps_checkout_dir, reponame, url, branch):
         command = ['git', 'clone', '-q', url, '-b', branch, checkout_dir]
         _logger.info('Calling %s', ' '.join(command))
         subprocess.check_call(command)
+    else:
+        command = ['git', '--git-dir=' + os.path.join(checkout_dir, '.git'),
```

Comment:
When using the `--git-dir` argument, you should use `--work-tree`, to define the working directory of your git repository.

Try calling `git --git-dir=/some/path/.git` from outside your git repository. The changes will be applied in your current working directory, instead of the git repository.